src/bindings/c/include/openvino/ov_common.h

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a common header file for the C API
+ *
+ * @file ov_common.h
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#ifdef __cplusplus
+#    define OPENVINO_C_API_EXTERN extern "C"
+#else
+#    define OPENVINO_C_API_EXTERN
+#endif
+
+#if defined(OPENVINO_STATIC_LIBRARY) || defined(__GNUC__) && (__GNUC__ < 4)
+#    define OPENVINO_C_API(...) OPENVINO_C_API_EXTERN __VA_ARGS__
+#    define OV_NODISCARD
+#else
+#    if defined(_WIN32)
+#        define OPENVINO_C_API_CALLBACK __cdecl
+#        ifdef openvino_c_EXPORTS
+#            define OPENVINO_C_API(...) OPENVINO_C_API_EXTERN __declspec(dllexport) __VA_ARGS__ __cdecl
+#        else
+#            define OPENVINO_C_API(...) OPENVINO_C_API_EXTERN __declspec(dllimport) __VA_ARGS__ __cdecl
+#        endif
+#        define OV_NODISCARD
+#    else
+#        define OPENVINO_C_API(...) OPENVINO_C_API_EXTERN __attribute__((visibility("default"))) __VA_ARGS__
+#        define OV_NODISCARD        __attribute__((warn_unused_result))
+#    endif
+#endif
+
+#ifndef OPENVINO_C_API_CALLBACK
+#    define OPENVINO_C_API_CALLBACK
+#endif
+
+#define MAX_DIMENSION 8
+
+typedef struct ov_core ov_core_t;
+typedef struct ov_output_const_node ov_output_const_node_t;
+typedef struct ov_output_node ov_output_node_t;
+typedef struct ov_model ov_model_t;
+typedef struct ov_compiled_model ov_compiled_model_t;
+typedef struct ov_infer_request ov_infer_request_t;
+
+typedef struct ov_property ov_property_t;
+
+typedef struct ov_tensor ov_tensor_t;
+typedef struct ov_rank ov_rank_t;
+typedef struct ov_dimension ov_dimension_t;
+typedef struct ov_dimensions ov_dimensions_t;
+typedef struct ov_layout ov_layout_t;
+typedef struct ov_partial_shape ov_partial_shape_t;
+
+typedef struct ov_preprocess_prepostprocessor ov_preprocess_prepostprocessor_t;
+typedef struct ov_preprocess_inputinfo ov_preprocess_inputinfo_t;
+typedef struct ov_preprocess_inputtensorinfo ov_preprocess_inputtensorinfo_t;
+typedef struct ov_preprocess_outputinfo ov_preprocess_outputinfo_t;
+typedef struct ov_preprocess_outputtensorinfo ov_preprocess_outputtensorinfo_t;
+typedef struct ov_preprocess_inputmodelinfo ov_preprocess_inputmodelinfo_t;
+typedef struct ov_preprocess_preprocesssteps ov_preprocess_preprocesssteps_t;
+
+/**
+ * @struct ov_shape_t
+ * @brief Reprents a static shape.
+ */
+typedef struct {
+    int64_t rank;
+    int64_t dims[MAX_DIMENSION];
+} ov_shape_t;
```

Comment:
I think it should be inside `ov_shape,h`

Reply:
ok